libraries/AP_Proximity/AP_Proximity_TeraRangerTower.cpp

```diff
@@ -94,7 +94,7 @@ void AP_Proximity_TeraRangerTower::update_sector_data(int16_t angle_deg, uint16_
 {   
     // Get location on 3-D boundary based on angle to the object
     const AP_Proximity_Boundary_3D::Face face = boundary.get_face(angle_deg);
-    if (distance_cm != 0xffff) {
+    if (distance_cm != 0xffff && !check_obstacle_near_land(angle_deg, distance_cm * 0.0001f)) {
```

Comment:
is there perhaps one too many zeros here?  I think maybe it should be 0.001f which is 1/1000.

Reply:
Rookie mistake! Thanks for catching it